README.rst

```diff
@@ -3,7 +3,26 @@ CNXML Library
 
 This is a python library for working with CNXML content, which includes the CNXML and COLLXML formats.
 
+Install
+-------
+
+This library requires Python >=3.5.
+
+Install using::
+
+   pip install -e "."
```

Comment:
Same as before, when you came across this in nebuchadnezzar... `pip install --upgrade pip setuptools`.